<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="txtIncognitas">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Inc칩gnitas</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtVarConocidas">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Variables Conocidas</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCalcIntermedios">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>C치lculos Intermedios</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTam">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Tama침o</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOfuscacion">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Ofuscaci칩n</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCalcular">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Calcular</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDificultad">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>390</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dificultad= </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEnunciado">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>271</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Enunciado</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpiar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Limpiar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
